Beitragsordnung_ZTL.tex

```diff
@@ -77,15 +78,15 @@ \section{Erstattungen}
 	\item Sollte dem Verein durch Zahlung des Beitrags per Lastschriftverfahren Kosten entstehen, die das Mitglied verursacht hat (z. B. keine ausreichende Kontodeckung, keine Änderungsmitteilung der Kontoverbindung, etc.) so sind diese vom Mitglied zu tragen und an den Verein zu erstatten.
 \end{enumerate}
 
-\section{Inkrafttreten}
+\section{Inkrafttreten}\label{inkrafttreten}
 \begin{enumerate}
-	\item Diese Beitragsordnung tritt am 01.09.2020 in Kraft.
+	\item Diese Beitragsordnung tritt am 11.09.2023 in Kraft.
 \end{enumerate}
 
 
 \vspace{2.5cm}
 
-\noindent Beschlossen bei der Mitgliederversammlung am 29.08.2020
+\noindent Geändert in \ref{beitraege}.\ref{beitraege-schulmitgliedschaft} (Höhe der Beiträge, neue Schulmitgliedschaft, Wegfall MEC-Mitgliedschaft) sowie \ref{inkrafttreten} in der Mitgliederversammlung am 09.09.2023. In Originalfassung bei der Mitgliederversammlung am 29.08.2020 beschlossen.
```

Comment:
Wozu der Changelog in der Beitragsordung? Ich sehe keine Notwendigkeit.

Reply:
Ist von Vorteil bei Fragen, was sich denn wann geändert hat.

Reply:
@casartar Und nu?